test/IntrusivePtr.cpp

```diff
@@ -0,0 +1,198 @@
+#include "pch.h"
+#include <kf/stl/new>
+#include <kf/boost/intrusive_ptr.hpp>
+#include <kf/boost/intrusive_ref_counter.hpp>
+#include <kf/UString.h>
```

Comment:
Let's minimize test dependencies and remove UString from the test.

Reply:
++